parachain/src/chain_spec.rs

```diff
@@ -356,7 +360,201 @@ fn testnet_genesis(
+                        "Band".as_bytes().to_vec(),
+                    ),
+                ],
+                id,
+                6,
+            )
+        },
+        Vec::new(),
+        None,
+        None,
+        Some(get_properties()),
+        Extensions {
+            relay_chain: "kusama".into(),
+            para_id: id.into(),
+        },
+    )
+}
+
+fn mainnet_genesis(
+    root_key: AccountId,
+    initial_authorities: Vec<AuraId>,
+    endowed_accounts: Vec<AccountId>,
+    authorized_oracles: Vec<(AccountId, Vec<u8>)>,
+    id: ParaId,
+    bitcoin_confirmations: u32,
+) -> GenesisConfig {
+    GenesisConfig {
+        system: SystemConfig {
+            code: WASM_BINARY
+                .expect("WASM binary was not build, please build it!")
+                .to_vec(),
+            changes_trie_config: Default::default(),
+        },
+        aura: AuraConfig {
+            authorities: initial_authorities,
+        },
+        aura_ext: Default::default(),
+        parachain_system: Default::default(),
+        parachain_info: ParachainInfoConfig { parachain_id: id },
+        sudo: SudoConfig {
+            // Assign network admin rights.
+            key: root_key.clone(),
+        },
+        tokens: TokensConfig {
+            balances: endowed_accounts.iter().cloned().map(|k| (k, KSM, 1 << 60)).collect(),
+        },
+        oracle: OracleConfig {
+            authorized_oracles,
+            max_delay: DEFAULT_MAX_DELAY_MS,
+        },
+        btc_relay: BTCRelayConfig {
+            bitcoin_confirmations,
+            parachain_confirmations: bitcoin_confirmations.saturating_mul(BITCOIN_BLOCK_SPACING),
+            disable_difficulty_check: false,
+            disable_inclusion_check: false,
+            disable_op_return_check: false,
+        },
+        issue: IssueConfig {
+            issue_period: DAYS,
+            issue_btc_dust_value: DEFAULT_DUST_VALUE,
+        },
+        redeem: RedeemConfig {
+            redeem_transaction_size: virtual_transaction_size(
+                TransactionInputMetadata {
+                    count: 2,
+                    script_type: InputType::P2WPKHv0,
+                },
+                TransactionOutputMetadata {
+                    num_op_return: 1,
+                    num_p2pkh: 2,
+                    num_p2sh: 0,
+                    num_p2wpkh: 0,
+                },
+            ),
+            redeem_period: DAYS,
+            redeem_btc_dust_value: DEFAULT_DUST_VALUE,
+        },
+        replace: ReplaceConfig {
+            replace_period: DAYS,
+            replace_btc_dust_value: DEFAULT_DUST_VALUE,
+        },
+        vault_registry: VaultRegistryConfig {
+            minimum_collateral_vault: vec![(CurrencyId::KSM, 0)],
+            punishment_delay: DAYS,
+            secure_collateral_threshold: vec![(CurrencyId::KSM, FixedU128::checked_from_rational(200, 100).unwrap())], /* 200% */
```

Comment:
Note that this is set to 100% for mainnet.

Reply:
please set to 300, thanks!